spec/controllers/sessions_controller_spec.rb

```diff
@@ -0,0 +1,71 @@
+require "rails_helper"
+
+describe SessionsController do
+  context "GET #new" do
+    context "without a password" do
+      it "does not set current user" do
+        user = create(:user)
+        session_params = { email: user.email }
+
+        post :create, params: { session: session_params }
```

Comment:
A suggestion instead of defining sessions_params for such a small hash, maybe just define them on this line. I.e.
```
post :create, params: { session: { email: user.email } }
```

Reply:
Done in sha: 8369fcd